src/components/elements/table/tableRowActionUtil.tsx

```diff
@@ -0,0 +1,44 @@
+import { Box } from '@mui/material';
+import { visuallyHidden } from '@mui/utils';
+import { ColumnDef } from '@/components/elements/table/types';
+import { clientBriefName, entryExitRange } from '@/modules/hmis/hmisUtil';
+import {
+  ClientDashboardRoutes,
+  EnrollmentDashboardRoutes,
+} from '@/routes/routes';
+import { ClientNameFragment, EnrollmentFieldsFragment } from '@/types/gqlTypes';
+import { generateSafePath } from '@/utils/pathEncoding';
+
+// TODO(#6761) - enforce header-less columns always have accessible text in GenericTable
+export const BASE_ACTION_COLUMN_DEF: ColumnDef<any> = {
+  header: <Box sx={visuallyHidden}>Actions</Box>,
+  key: 'Actions',
+  tableCellProps: { sx: { p: 0 } },
+  render: '', // gets overridden when used
+};
+
+export const getViewClientAction = (client: ClientNameFragment) => {
```

Comment:
Naming suggestion: `getViewClientMenuItem` and `getViewEnrollmentMenuItem` since these are returning `CommonMenuItem` objects